flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/plan/util/PlanConfigUtil.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.plan.util;
+
+import org.apache.flink.table.api.TableConfig;
+import org.apache.flink.table.api.TableConfigOptions;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * Utility for configurations relative planner.
+ */
+public class PlanConfigUtil {
+
+	/**
+	 * Returns true if given {@link OperatorType} is enabled, else false.
+	 */
+	public static boolean isOperatorEnabled(TableConfig config, OperatorType operator) {
```

Comment:
I think this can be an interface from TableConfig, and no need to introduce a util class now.